src/shortestpaths/dijkstra.jl

```diff
@@ -116,3 +116,72 @@ end
 
 dijkstra_shortest_paths(g::AbstractGraph, src::Integer, distmx::AbstractMatrix = DefaultDistance(); allpaths=false, trackvertices=false) =
 dijkstra_shortest_paths(g, [src;], distmx; allpaths=allpaths, trackvertices=trackvertices)
+
+
+
+
+"""
+    struct MultipleDijkstraState{T, U}
+
+An [`AbstractPathState`](@ref) designed for multisource_dijkstra_shortest_paths calculation.
+"""
+struct MultipleDijkstraState{T, U<:Integer}<:AbstractPathState
+    dists::Matrix{T}
+    parents::Matrix{U}
+end
+
+@doc_str """
+    multisource_dijkstra_shortest_paths(g, sources=vertices(g), distmx=DefaultDistance())
+    parallel_multisource_dijkstra_shortest_paths(g, sources=vertices(g), distmx=DefaultDistance())
+
+This function computes the shortest paths between all pairs of vertices in graph `g` by running
```

Comment:
The docstring should be active voice:
"Compute the shortest paths .... Return ____"